packages/airview-compliance-ui/src/features/application-tile/application-tile-content.js

```diff
@@ -74,7 +72,7 @@ function useApplicationTileContentStyles() {
       padding: 0,
     },
     collapseWrapperInner: {
```

Comment:
SX API refactor is preventing these styles being applied to the correct DOM node. Can you update to the example below, which will fix the layout error in the UI.


```jsx
collapseWrapperInner: {
   "& .MuiCollapse-wrapperInner": {
      paddingTop: collapsible ? 2 : 0,
    },
 }
```

Reply:
thank you so much! it is actually working now

Reply:
Welcome :)